Services/UI/OJS.Services.Ui.Models/Contests/ContestParticipationServiceModel.cs

```diff
@@ -30,10 +30,6 @@ public class ContestParticipationServiceModel : IMapExplicitly
     public void RegisterMappings(IProfileExpression configuration)
         => configuration.CreateMap<Participant, ContestParticipationServiceModel>()
             .ForMember(d => d.Contest, opt => opt.MapFrom(s => s.Contest))
-            .ForMember(d => d.LastSubmissionTime, opt => opt.MapFrom(s =>
-                s.Submissions.Any()
-                    ? (DateTime?)s.Submissions.Max(x => x.CreatedOn)
-                    : null))
```

Comment:
Is this not working? It should work without lazy loading...

Reply:
I removed it because in the cases it is being used we are mapping from a participant which doesn't have it's submissions include and therefore s.Submissions.Any() returns false.

Reply:
If the mapping is done on IQueryable<>, `.Include` is not needed.

Reply:
This model is used in StartContestParticipation method where the mapping is done from a Participant entity, not from IQueryable<>. This is the original way the method was developed and therefore mapping doesn't work if the participant's Submissions aren't included. 
Should i refactor the data query to return IQueryable<> rather than a Participant entity.

Reply:
Maybe in another issue.